Geovalidation.md

```diff
@@ -0,0 +1,143 @@
+#Geovalidation API
+
+&nbsp;
+
+#Contents
+
+- [Description](#description)
+- [Request Structure](#request-structure)
+- [Response Structure](#response-structure)
+- [Versioning](#versioning)
+
+----------------------------
+#Description
+
+Accepts structured or unstructured location information and tests the validity of the resulting resolved geographic coordinate based on the strategies requested and the location types accepted. This service supports the HTTP/GET method only.
```

Comment:
should just be "structured", this service doesn't do unstructured

"and tests the validity of the resulting resolved geographic coordinate"

this is worded kind of awkwardly (and isn't exactly true, we don't validate the coordinate), how about something like:

"Accepts structured location data and attempts to geocode the data using the validation strategies specified in the request. Each successful geocoding call is verified against the list of acceptable location types specified in the request. The outcome of each strategy is described in the response, along with any normalized data obtained through the geocoding calls.